### PR TITLE
Check if value exists on file like fields

### PR DESCRIPTION
**Pull request by agitator**

### CODE REVIEW

File: collective/jsonify/wrapper.py [hunk 1]
```diff
@@ -310,8 +310,8 @@ def get_archetypes_fields(self):
                 value = self._get_at_field_value(field)
                 value2 = value
 
-                if not isinstance(value, str):
-                    if isinstance(value.data, str):
+                if value and not isinstance(value, str):
+                    if value and isinstance(value.data, str):
```
Comment (hvelarde) [inline collective/jsonify/wrapper.py]: I think this should be `if value.data and isinstance(value.data, str):`

Comment (agitator) [reply collective/jsonify/wrapper.py]: I got a error on value.data because value was none, while exporting a site that was using CMFBibliography.

**Comment by agitator** [comment]
I got a error on value.data because value was none, while exporting a site that was using CMFBibliography.

File: collective/jsonify/wrapper.py (hunk 1 above)
Comment (mauritsvanrees) [reply collective/jsonify/wrapper.py]: Yes, in the second `if` the `value` is already guaranteed to be true.
Maybe check if `value` has an attribute `data`.

Comment (thet) [reply collective/jsonify/wrapper.py]: like
```
if value and not isinstance(value, str):
    if isinstance(getattr(value, 'data', None), str):
```

**Comment by agitator** [comment]
updated

**Pull request merged**